packages/nuxt/src/core/plugins/resolve-deep-imports.ts

```diff
@@ -34,7 +34,7 @@ export function resolveDeepImportsPlugin (nuxt: Nuxt): Plugin {
       conditions = [...resolvedConditions]
 
       const { runtimeDependencies = [] } = await tryImportModule<typeof import('nitro/runtime/meta')>('nitro/runtime/meta', {
-        url: nuxt.options.modulesDir.map(d => directoryToURL(d)),
+        url: new URL(import.meta.url),
```

Comment:
We have fastpath in exsolve BTW: https://github.com/unjs/exsolve/blob/main/src/resolve.ts#L340